src/excmds.ts

```diff
@@ -1678,13 +1678,32 @@ if (fullscreenApiIsPrefixed) {
 }
 // }
 
+/** Expands all occurrences of "<.*>" to matching args.
+   @hidden
+ */
+
+//#content_helper
+export function parseaucmdargs(aucmd: string, args: AutocmdArgs): string {
+    const re = /^\"<(\w+)>\"$/
```

Comment:
How hard would it be to factor this out to into a well-contained library so you can add some simple test cases? It _looks_ right, and I expect that Pentadactyl collected enough real-world observations to establish that strings like `<file>` and such didn't occur that often in URLs, but I'd prefer to not start accidentally replacing random other `<....>` pairs.

Reply:
I'm all for factoring it out into a library, the problem there would be the implementation. If we remove loadaucmds from excmds along with the parser I'm guessing it'd have to live in the content script somehow similar to what autocontainers does in the background.

The current implementation is pretty heavy handed so I'm nigh sure you won't get it mixed with urls or other things. I split the aucmd string on spaces and check for exact matches on `^"<WORD>"$`. The problem with this is that you can't combine params, `"<file> <url>"` will not be matched since the split on space would create `"<file>` and `<url>"` which won't match the regex.